### PR TITLE
chore: typescript clean up

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request makes code changes to address TypeScript errors.

image
😄 

### Approvals

- [x] Dev

### CODE REVIEW

File: centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx [hunk 1]
```diff
@@ -175,8 +174,8 @@ function InvestRedeemInput({ defaultView: defaultViewProp }: InputProps) {
                 `${state.trancheCurrency?.name} is only available to Non-U.S. persons.`
               ) : (
                 <>
-                  ({metadata?.pool?.issuer?.name} tokens are available to U.S. and Non-U.S. persons. U.S. persons must
```
Comment (jpangelle) [inline centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx:178]: stray opening parenthesis

File: centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx [hunk 2]
```diff
@@ -175,8 +174,8 @@ function InvestRedeemInput({ defaultView: defaultViewProp }: InputProps) {
...
-                  be verified “accredited investors”.{' '}
```
Comment (jpangelle) [inline centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx:179]: weirdly formatted quotation marks

File: centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx [hunk 3]
```diff
@@ -218,7 +217,7 @@ function Header() {
               {formatBalance(state.investmentValue, state.poolCurrency?.symbol, 2, 0)}
             </TextWithPlaceholder>
           </Stack>
-          {/*
```
Comment (jpangelle) [inline centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx:221]: unnecessary space

File: centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx [hunk 4]
```diff
@@ -749,7 +748,6 @@ function PendingOrder({
   onChangeOrder: () => void
 }) {
   const { state } = useInvestRedeem()
-  const { message: epochTimeRemaining } = useEpochTimeCountdown(pool.id!)
```
Comment (jpangelle) [inline centrifuge-app/src/components/InvestRedeem/InvestRedeem.tsx:752]: unused variable

File: centrifuge-app/src/components/IssuerSection.tsx [hunk 5]
```diff
@@ -18,11 +15,6 @@ type IssuerSectionProps = {
 export function IssuerSection({ metadata }: IssuerSectionProps) {
   const cent = useCentrifuge()
   const [isDialogOpen, setIsDialogOpen] = React.useState(false)
-  const { pid: poolId } = useParams<{ pid: string }>()
-  const isTinlakePool = poolId.startsWith('0x')
-  const pool = usePool(poolId)
-  const network = isTinlakePool ? ((pool as TinlakePool).network === 'goerli' ? 5 : 1) : 'centrifuge'
-  const explorer = useGetExplorerUrl(network)
```
Comment (jpangelle) [inline centrifuge-app/src/components/IssuerSection.tsx:25]: `explorer` variable was unused and so the variables above were also unused

File: centrifuge-app/src/components/Tooltips.tsx [hunk 6]
```diff
@@ -265,7 +265,7 @@ export type TooltipsProps = {
   variant?: 'primary' | 'secondary'
   label?: string | React.ReactNode
   props?: any
-} & TextProps
+} & Partial<Pick<TextProps, 'style' | 'body'>>
```
Comment (jpangelle) [inline centrifuge-app/src/components/Tooltips.tsx:268]: This fixes dozens of type errors for the `Tooltips` component

File: centrifuge-js/src/modules/pools.ts [hunk 7]
```diff
@@ -710,6 +710,8 @@ type InvestorTransaction = {
   tokenAmount: CurrencyBalance | undefined
   tokenPrice: Price | undefined
   transactionFee: CurrencyBalance | null
+  chainId: number
+  evmAddress?: string
```
Comment (jpangelle) [inline centrifuge-js/src/modules/pools.ts:714]: This was never added when it should have been.

**Review by sophialittlejohn — APPROVED** [review]
🔥

**Pull request merged**